src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_VALUE = 100;
+    private static final int DEFAULT_CIRCLE_RADIUS = 10;
+    private static final Color DEFAULT_CIRCLE_COLOR = Color.WHITE;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public int getValue() {
+        return random.nextInt(MAX_VALUE);
+    }
```

Comment:
Let's use proper access modifiers for method. If you don't use this method outside of the class, it should be private

Reply:
Also, move private methods to the end of the class;)